pkg/cmd/tracee.go

```diff
@@ -97,12 +97,19 @@ func (r Runner) Run(ctx context.Context) error {
 	return t.Run(ctx) // return when context is cancelled by signal
 }
 
-func PrintEventList() {
+func PrintEventList(printRulesSet bool) {
 	padChar, firstPadLen, secondPadLen := " ", 9, 36
 	titleHeaderPadFirst := getPad(padChar, firstPadLen)
 	titleHeaderPadSecond := getPad(padChar, secondPadLen)
 
 	var b strings.Builder
+
+	if printRulesSet {
+		b.WriteString("Events: " + titleHeaderPadFirst + "Sets:" + titleHeaderPadSecond + "Arguments:\n")
+		b.WriteString("____________  " + titleHeaderPadFirst + "____ " + titleHeaderPadSecond + "_________" + "\n\n")
+		printEventGroup(&b, events.StartRulesID, events.MaxRulesID)
```

Comment:
This is only relevant if `printRulesSet` is true.
I think we should take out these small blocks of
```
b.WriteString
b.WriteString
printEventGroup
```
into their own small function then we can use these blocks to create a PrintList function for each of these cases.

Reply:
I wasn't worrying about it because the idea is to stop supporting `tracee-ebpf` binary, so soon we will have only a version of it. Though I don't mind the refactoring if you think it is important, I can create a refactor issue as a follow-up.